common/src/main/java/uk/gov/justice/services/common/converter/ObjectToJsonObjectConverter.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.services.common.converter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javax.inject.Inject;
+import javax.json.JsonObject;
+import java.io.IOException;
+
+/**
+ * Converts a Pojo to a JsonObject
+ */
+public class ObjectToJsonObjectConverter implements Converter<Object, JsonObject> {
+
+    @Inject
+    ObjectMapper mapper;
+
+    @Override
+    public JsonObject convert(final Object source) {
+        try {
+            return mapper.readValue(mapper.writeValueAsString(source), JsonObject.class);
```

Comment:
Wrap the response in an Optional?
Would be Converter<Object,Optional<JsonObject> - Might need to change interface IF we want to do this..